### PR TITLE
CheatSheets: Update filenames and/or IDs to match

### PR DESCRIPTION
**Pull request by moollaza**
I've also updated IA Page ID's accordingly.

cc/ @zachthompson

### CODE REVIEW

File: share/goodie/cheat_sheets/json/golang.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "id": "go_lang_script_cheat_sheet",
+    "id": "golang_script_cheat_sheet",
```
Comment (zachthompson) [inline share/goodie/cheat_sheets/json/golang.json]: Mismatch here still

**Comment by zachthompson** [comment]
Looks like we have a new mismatch as well - [windows10.json](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/windows10.json#L2).

**Comment by moollaza** [comment]
@zachthompson I believe everything is fixed now :+1:

**Comment by zachthompson** [comment]
@moollaza 
```
Did not find angularjs_cheat_sheet in metadata (different id)
Did not find js_keycodes_cheat_sheet in metadata (no perl_module)
Did not find rose_meanings_cheat_sheet in metadata (no perl_module)
```

**Comment by zachthompson** [comment]
ping

**Comment by moollaza** [comment]
@zachthompson should be fixed now. Updated with recent master commits. I couldn't see anything wrong with js-keycodes.json

**Comment by zachthompson** [comment]
Js keycodes and rose didn't have `DDG::Goodie::CheatSheets` specified as their module on IA pages.  Something up with the angular and rose json it looks like.

**Comment by moollaza** [comment]
@zachthompson ah, thanks for catching that. Forgot to run the tests. Should be fixed now.

**Comment by zachthompson** [comment]
Nice :+1:

**Pull request merged**